tamr_toolbox/data_io/manage_dataset.py

```diff
@@ -0,0 +1,178 @@
+from typing import List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.dataset.resource import Dataset, DatasetSpec
+from tamr_unify_client.attribute.resource import AttributeSpec
+from tamr_unify_client.attribute.type import AttributeType
+
+
+def create(
+    *, tamr: Client, dataset_spec: DatasetSpec, attributes: Union[List[str], List[AttributeSpec]],
+) -> Dataset:
+    """Flexibly create a source dataset in Tamr. Will use array string as deafult attribute type
```

Comment:
Typo
```suggestion
    """Flexibly create a source dataset in Tamr. Will use array string as default attribute type
```

Reply:
Thanks Sam! I just pushed a commit that fixes the docstrings. I should have double checked that before marking it ready to review. Can you double check you are on the latest commit?